app/views/spree/user_registrations/new.html.erb

```diff
@@ -32,7 +32,7 @@
                      </nav>
                      <h1 class="title mb-0">Registration</h1>
                      <div class="sm-title mb-20" id="password-credentials" >It looks like you’re new here. We need a bit more info...</div>
-                     <%= render :partial => 'spree/shared/user_form', :locals => { :f => f } %>
+                     <%= render :partial => 'spree/shared/user_form', :locals => { :f => f , remember_me_field: true} %>
```

Comment:
remove this locale: remember_me_field: true. because we are not using remember me field